Packs/AzureActiveDirectory/Integrations/AzureADIdentityProtection/AzureADIdentityProtection.py

```diff
@@ -0,0 +1,261 @@
+from CommonServerPython import *
+import urllib3
+
+urllib3.disable_warnings()
+
+OUTPUTS_PREFIX = "AADIdentityProtection"
+BASE_URL = 'https://graph.microsoft.com/beta'
+REQUIRED_PERMISSIONS = (
+    'offline_access',  # allows device-flow login
+    'IdentityRiskEvent.Read.All',
+    'IdentityRiskyUser.ReadWrite.All'
+)
+
+
+def parse_list(raw_response: dict, human_readable_title: str, context_path: str) -> CommandResults:
+    """
+    converts a response of Microsoft's graph search into a CommandResult object
+    """
+    values = raw_response.get('value', [])
+    readable_output = tableToMarkdown(f'{human_readable_title.title()} ({len(values)} results)',
+                                      values,
+                                      removeNull=True,
+                                      headerTransform=pascalToSpace,
+                                      first_headers=['Id', 'userId', 'userPrincipalName', 'userDisplayName',
+                                                     'ipAddress', 'detectedDateTime'])
+    outputs = {f'{OUTPUTS_PREFIX}.{context_path}(val.id === obj.id)': values}
+
+    # removing whitespaces so they aren't mistakenly considered as argument separators in CLI
+    next_link = raw_response.get('@odata.nextLink', '').replace(' ', '%20')
+    if next_link:
+        next_link_key = f'{OUTPUTS_PREFIX}.NextLink(val.Description === "{context_path}")'
+        next_link_value = {'Description': context_path, 'URL': next_link}
```

Comment:
This is equivalent to `true == true` since you are running the query on the result item and not the context item. 

`val` is the result, while `obj` is the existing  `NextLink` in the Context. Switch it to `obj.Description` and make sure it works ok.